rs/ledger_suite/icrc1/ledger/tests/tests.rs

```diff
@@ -465,14 +449,15 @@ fn icrc1_test_upgrade_serialization() {
         upgrade_args,
         minter,
         true,
+        true,
     );
 }
 
 #[test]
 fn icrc1_test_upgrade_serialization_fixed_tx() {
     ic_ledger_suite_state_machine_tests::icrc1_test_upgrade_serialization_fixed_tx(
         ledger_mainnet_wasm(),
-        ledger_wasm(),
+        ledger_wasm_lowupgradeinstructionlimits(),
```

Comment:
Should this test (and the helper in `ic_ledger_suite_state_machine_tests`) be renamed to something more descriptive, since it takes a `lowupgradeinstructionlimits` wasm, and performs some checks that verify that multiple timer invocations are executed per migration?